jetstream.md

```diff
@@ -0,0 +1,121 @@
+
+> **NOTE: WINDOWS users** will need to install a UNIX-ready terminal to SSH-login (if you haven't already). We recommend Git bash. See workshop setup page for instructions.
+
+# Login & Launch Instance
+
+- **Login to [Jetstream](https://use.jetstream-cloud.org/application/jetstream/images/search) by clicking the "login" button towards the right-upper corner of the screen.**
+
+![](jetstream/images/login1.png)
+![](jetstream/images/login2.png)
+
+- **Fill in your Jetstream username and password and click "LOGIN"**
+
+![](jetstream/images/login3.png)
+
+- **Select the "Projects" tab and then click the "CREATE NEW PROJECT" button**
+
+![](jetstream/images/login4.png)
+
+- **Give your Project folder a name (Description is optional). Then click "CREATE".**
+
+![](jetstream/images/login5.png)
+
+- **Click on your newly created project and then Click on "NEW" and then "Instance" from the drop-down menu to start up a new virtual machine.**
+
+![](jetstream/images/login6.png)
+
+- **To select an image click on "Show All" tab and Search for "ANGUS 2019" and choose the "ANGUS 2019" image created by 'titus'.**
+
+![](jetstream/images/login7.png)
+
+Here, "image" refers to the resources that are pre-loaded into your computing workspace on your instance. Think of it like apps that come with your phone before you add new ones on your own.
+
+**You will be presented with options to choose and configure your virtual machine here:
+
+	+ Instance Name: e.x., "ANGUS 2019 base image" or you can leave it default which is the image name.
+
+	+	Base Image Version: "1.0"
+
+	+	Project: select your project folder
+
+	+	Allocation Source: DIBSI Jetstream account
+
+	+	Provider: "Jetstream - TACC"
+
+	+ Instance size: We recommend ""m1.medium" (CPU: 6, Mem: 16GB, Disk: 60GB)"for this tutorial; Though  depending on your allocations, choose most suitable one.**
+
+![](jetstream/images/login8.png)
+
+- **Launch instance and wait for the build to be deployed (~ 5-10 minutes).**
+
+> Note: During the build process: `scheduling-->building-->spawning-->deploying-->Networking-->N/A`; Be patient! Don't reload!. Once the virtual machine is ready, the "Activity" column will show "N/A" and the "Status" column will turn green and "Active".
+
+![](jetstream/images/login9.png)
+
+- **Navigate back to 'Projects' and click on your new instance's name to see more information related to the instance you just created! and Copy the IP address of your instance created.**
+
+![](jetstream/images/login10.png)
+
+> **Great! We have now built our very own remote virtual machine with all the software pre-installed. Next we will use SSH-Secure-Login to access these remote instances from our laptop !!!.**
+
+# SSH Secure-Login
+
+- MACOS & LINUX users can open a Terminal window now.
+- Windows users start a new session in git bash
```

Comment:
i think if they have windows 10, using the ubuntu distribution is a lot better than gitbash bc of `ming64` user things